src/crypto/subtle.cr

```diff
@@ -10,7 +10,7 @@ module Crypto::Subtle
   def self.constant_time_compare(x, y) : Bool
     x = x.to_slice
     y = y.to_slice
-    return false if x.size != y.size
+    raise ArgumentError.new("Compared strings have different lengths") if x.size != y.size
```

Comment:
Let's be positive, and since we can compare anything that responds to `to_slice` we could say:
```crystal
raise ArgumentError.new("Arguments must be equal length") unless x.size == y.size
```